src/spo/spoplus_loss.jl

```diff
@@ -1,15 +1,15 @@
 """
-    SPOPlusLoss{F}
+    SPOPlusLoss <: AbstractLossLayer
```

Comment:
Should this file belong in the `imitation` folder ?

Reply:
Perhaps yeah